cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -238,10 +238,10 @@ func RemoveOldControlPlaneLabel(client clientset.Interface) error {
 	return nil
```

Comment:
`RemoveOldControlPlaneLabel` is not used any more and can be removed as https://github.com/kubernetes/kubernetes/pull/110095/files#diff-fd7aac3195ba96f8f6beb0a559e48ebbc93ef87df4c197de70cacf9a97117c56L162.

Reply:
This is for nit. 
/lgtm

Reply:
I missed that. It has a TODO linking to the issue, at least.